Sources/BraintreeVenmo/BTVenmoAccountNonce.m

```diff
@@ -23,9 +28,17 @@ - (instancetype)initWithPaymentMethodNonce:(NSString *)nonce
 }
 
 - (instancetype)initWithPaymentContextJSON:(BTJSON *)paymentContextJSON {
-    return [[self.class alloc] initWithPaymentMethodNonce:[paymentContextJSON[@"data"][@"node"][@"paymentMethodId"] asString]
-                                                 username:[paymentContextJSON[@"data"][@"node"][@"userName"] asString]
-                                                isDefault:NO];
+    BTVenmoAccountNonce *accountNonce = [[self.class alloc] initWithPaymentMethodNonce:[paymentContextJSON[@"data"][@"node"][@"paymentMethodId"] asString] username:[paymentContextJSON[@"data"][@"node"][@"userName"] asString] isDefault:NO];
```

Comment:
```suggestion
    BTVenmoAccountNonce *accountNonce = [[self.class alloc] initWithPaymentMethodNonce:[paymentContextJSON[@"data"][@"node"][@"paymentMethodId"] asString] 
                                                                              username:[paymentContextJSON[@"data"][@"node"][@"userName"] asString] isDefault:NO];
```

Nitpick to new line at the `username:` param and line up the `:`. (GitHub suggestion formatting werid)